ServiceConfig.cs

```diff
@@ -32,6 +38,12 @@ public ServiceConfig(string filename)
             }
             var decoded = Encoding.UTF8.GetString(Convert.FromBase64String(base64EncodedAccessKey));
             AccessKey = JsonConvert.DeserializeObject<AccessKey>(decoded);
+
+            Username = Environment.GetEnvironmentVariable("LFDS_USERNAME");
+            Password = Environment.GetEnvironmentVariable("LFDS_PASSWORD");
+            BaseUrl = Environment.GetEnvironmentVariable("SELFHOSTED_REPOSITORY_API_BASE_URI");
+            Organization = Environment.GetEnvironmentVariable("LFDS_ORGANIZATION");
+            AuthorizationType = Environment.GetEnvironmentVariable("AUTHORIZATION_TYPE");
```

Comment:
When no AuthorizationType is in the environment, a null reference exception is thrown. Maybe just throw an exception if this value is not set
![image](https://user-images.githubusercontent.com/102689343/174308201-aef01c5d-3d43-46eb-acc4-e817be299039.png)